src/main/java/com/dongtronic/diabot/commands/nightscout/NightscoutCommand.kt

```diff
@@ -101,27 +99,15 @@ class NightscoutCommand(category: Command.Category) : DiabotCommand(category, nu
         }
 
         val shortReply = NightscoutDAO.getInstance().listShortChannels(event.guild.id).contains(event.channel.id)
-        var response: ReceivedMessage? = null // dirty workaround
 
         if (shortReply) {
             val message = buildShortResponse(dto, userDTO.displayOptions)
-            event.reply(message) { response = it as ReceivedMessage }
+            event.reply(message) { addReactions(dto, it) }
```

Comment:
since we've got two variables now, perhaps rename `it` to `replyMessage` or something similar.